app/controllers/landing_controller.rb

```diff
@@ -1,4 +1,6 @@
 class LandingController < ApplicationController
   def index
+    @disable_nav = true
```

Comment:
What does this do?

Reply:
Under views/layouts, you'll see I made a _header.html.erb that just has a link to the home page. This gets rendered in layouts/application.html.erb so that every page has that link at the top by default: `<%= render "layouts/header" unless @disable_nav %>`. The `unless @disable_nav` allows you to put `@disable_nav = true` in any controller you don't want the link to be rendered in. I didn't want there to be a link to the home page on the home page itself, so I put  `@disable_nav = true` in its controller.

Reply:
Sweet! Belay on!